elastic/kibana
# [Infra UI] Implement Metric charts section

Closes [#162942](https://github.com/elastic/kibana/issues/162942)

## Summary

This PR adds new metrics to the asset details page: 
- CPU usage (Broken down)
- Load (Broken down)
- Memory Usage (Broken down)
- Disk Usage by Mount Point

The hosts view flyout metrics remain the same as before.

## Testing 
⚠️ I would suggest running metricbeat in Docker as some of the metrics used in `CPU usage (Broken down)` are not available in MacOs (in discussion with beats team currently)
- Go to hosts page
- Open the flyout for a host
- Click open as page link
- Check the new metrics on different screen size (the labels should be visible as shown: 

![Image](https://github.com/elastic/kibana/assets/14139027/6d98996b-2eb1-414b-92a2-870932ad4284)

![Image](https://github.com/elastic/kibana/assets/14139027/e612c7b4-8d93-4c42-883b-fc5e620ab2a0)
